source/vistas/core/timeline.py

```diff
@@ -7,6 +7,8 @@
 
 
 class Timeline:
+    """ A represenation of a temporal timeline. """
```

Comment:
Comments are good, except when they're not. They should always be run through the filter of "does this explain something not immediately apparent?". If your class is called `Timeline` and your comment is "This is a timeline", the comment is at best unnecessary and at worse a future liability when functionality changes and the comment doesn't.